accelerometer:
  noise_std:
    x: 45.47744073713912
    y: 39.81367624322075
    z: 60.32319301893758
  offset:
    x: 1236.372
    y: 299.428
    z: -80.7
duration_seconds: 10
gyroscope:
  noise_std:
    x: 34.197776521288624
    y: 22.817437104109654
    z: 23.568535296025505
  offset:
    x: -270.741
    y: 320.758
    z: 62.988
timestamp: '2025-11-17T03:53:43.110908'
total_samples: 1000
